exercises/tom_clayton/ex02_oo_basics/src/ShapeSorter.h

```diff
@@ -0,0 +1,40 @@
+#ifndef SHAPESORTER_H_
+#define SHAPESORTER_H_
+
+#include "Shape.h"
+#include "Square.h"
+#include "Rectangle.h"
+#include "Circle.h"
+#include "Triangle.h"

```

Comment:
You should not need to include the headers for all the shapes - just the `Shape.h` class. Try to avoid including any unnecessary imports as for large projects like mantid, this can slow the compile time